megatron/core/inference/contexts/dynamic_context.py

```diff
@@ -682,15 +683,26 @@ def from_config(
         """
         # TODO: Add other necessary configs from inference_config
 
-        # Max sequence length. (todo: why does inference config have default value
-        # for inference_max_seq_length?)
-        model_max_seq_len = model.get_max_sequence_length()
+        # Max sequence length.
+        def get_model_attr(key):
+            try:
+                return getattr(model, key)
+            except Exception as e:
+                raise e
+
+        position_embedding_type = get_model_attr("position_embedding_type")
```

Comment:
Can you use `get_attr_wrapped_model` from `megatron/core/utils.py`? https://github.com/NVIDIA/Megatron-LM/blob/d2bd9fa89fe2fb74f002d68ad17cd5e47181dfcc/megatron/core/utils.py#L575